handler/additional_types.go.template

```diff
@@ -0,0 +1,22 @@
+// This file is generated with go generate. Any changes to it will be lost after
+// subsequent generates.
+// If you want to edit it go to types.go.template
+
+package handler
+
+import (
+	"github.com/ironsmile/nedomi/config"
+	"github.com/ironsmile/nedomi/types"
+
+    {{range .}}"{{.}}"
+{{end}}
+)
+
+var _ = func() (s struct{}){
```

Comment:
Is there a particular reason why this must be a function call? Wouldn't it work just the same if it was in the global scope?